src/components/main-menu/main-menu.js

```diff
@@ -1,21 +1,56 @@
-import React from 'react'
+import React, { useRef } from 'react'
 import { Link } from 'gatsby';
 import PropTypes from 'prop-types'
 
-const MainMenu = ({ redHeader }) => (
-  <div className='header-nav'>
-    <Link to='/'>Company</Link>
-    <Link to='/services'>Services</Link>
-    <Link to='/case-studies/'>Our work</Link>
-    <a href='https://medium.com/civicactions'>Insights</a>
-    <Link to='/careers/'>Careers</Link>
-    { redHeader ?
-      <Link to='/contact' className='primary-button'>Contact us</Link>
-      :
-      <Link to='/contact'><div className='primary-button'>Contact us</div></Link>
+const MainMenu = ({ redHeader }) => {
+
+  const headingsRef = useRef([]);
+  const onCollapseClick = (index) => {
+    const current = headingsRef.current[index].className;
+    headingsRef.current[index].className = current.includes('expanded')
+      ? 'menu--list__item'
+      : 'menu--list__item expanded';
+
+    if (current.includes('expanded')) {
+      headingsRef.current[index].setAttribute('aria-expanded', 'false')
+    } else {
+      headingsRef.current[index].setAttribute('aria-expanded', 'true')
     }
-  </div>
-)
+  };
+
+  return (
+    <nav className='header-nav'>
+      <ul className="menu--list">
+        <li
+          className='menu--list__item'
+          ref={(e) => (headingsRef.current[0] = e)}>
+          <button
+            className="heading"
+            onKeyDown={() => { }}
+            tabIndex={0}
+            onClick={() => onCollapseClick(0)}
+          >
+            Company
+          </button>
+          <ul>
+            {/*<li><Link to="/">About</Link></li>*/}
+            <li><Link to="/team">Team</Link></li>
+            <li><Link to="/press">Press</Link></li>
+          </ul>
+        </li>
+        <li className='menu--list__item'><Link to='/services'>Services</Link></li>
+        <li className='menu--list__item'><Link to='/case-studies/'>Our work</Link></li>
```

Comment:
Maybe check with Melinda, to see if any other menu links hold child item should show a drop down?